src/balena-model.ts

```diff
@@ -253,7 +254,7 @@ export interface Application {
 	uuid: string;
 	is_public: boolean;
 	application__has__env_var_name?: ApplicationEnvironmentVariable[];
-	application__has__name?: ApplicationConfigVariable[];
+	application__has__config_var_name?: ApplicationConfigVariable[];
```

Comment:
Do we not have a test that verifies this file is up to date?

Reply:
Looks like we don't. Let me add one while this gets merged.
Moreover I think that the generated property name is not correct either and that abstract-sql-to-typscript should have been using the Term Form when available, right?

Reply:
Actually, we do have a test in bC, so let me copy it here.